internal/manifests/distributor/distributor.go

```diff
@@ -90,6 +90,26 @@ func deployment(params manifestutils.Params) *v1.Deployment {
 									ContainerPort: manifestutils.PortMemberlist,
 									Protocol:      corev1.ProtocolTCP,
 								},
+								{
```

Comment:
I think we should expose these ports in the service only if the gateway is disabled. I know tempo won't listen on these ports based on the config, but I think the service should also mirror this.

Reply:
Done!